@xen-orchestra/openflow/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "private": true,
```

Comment:
Will this module be used as a dependency?

I so, it should probably be published to the npm registry, therefore you can set this entry to false.

Reply:
It's included in the sdn-controller plugin yes.